intermediate_source/ddp_tutorial.rst

```diff
@@ -79,12 +80,20 @@ be found in
 
 
     def setup(rank, world_size):
-        os.environ['MASTER_ADDR'] = 'localhost'
-        os.environ['MASTER_PORT'] = '12355'
-
-        # initialize the process group
-        dist.init_process_group("gloo", rank=rank, world_size=world_size)
+        if sys.platform == 'win32':
+            # Distributed package only covers collective communications with Gloo backend and FileStore
```

Comment:
nit: below is how the built doc looks like. Shall we break the lines into shorter ones to avoid line wrapping?

<img width="852" alt="Screen Shot 2020-10-12 at 10 59 03 AM" src="https://user-images.githubusercontent.com/16999635/95761238-05ee9f80-0c7a-11eb-870d-013e6ad8cac9.png">


```
# Distributed package only covers collective communications with 
# Gloo backend and FileStore on Windows platform. Set init_method 
# parameter in init_process_group to a local file.
# Example init_method="file:///f:/libtmp/some_file"
...

dist.init_process_group(
    "gloo", 
    init_method=init_method, 
    rank=rank, 
    world_size=world_size
)
```

Reply:
Yes, we should wrap it. Just resolved.